packages/core/cli/start.js

```diff
@@ -3,7 +3,7 @@
 const getEnv = require('../config/env');
 const {
   API_ROOT_URL,
-  API_ORIGN,
+  API_ORIGIN,
```

Comment:
This has bothered me every time I see it haha, glad it's fixed :)

Reply:
haha I didn't even realize I misspelled it until now 😬